testing/trino-faulttolerant-tests/pom.xml

```
@@ -10,6 +10,7 @@
     </parent>
 
     <artifactId>trino-faulttolerant-tests</artifactId>
+    <description>Trino - Fault Tolerant Execution tests</description>
```

```suggestion
    <description>Trino - Fault-tolerant execution tests</description>
```

See https://trino.io/docs/current/admin/fault-tolerant-execution.html